tests/qiskit_qir/test_basic_gates.py

```
@@ -0,0 +1,518 @@
+
+def _assert_body_ops(module, expected_ops):
+    """Assert that the QIR body contains exactly the expected operations in order.
+
+    Args:
+        module: PyQIR Module.
+        expected_ops: List of expected QIR instruction strings.
+    """
+    body = _get_body(module)
+
+    # Filter to only "call" lines and "ret" for comparison
+    gate_lines = [l for l in body if l.startswith("call void @__quantum__qis__")]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)